apps/site/reducers/__tests__/releaseReducer.test.mjs

```diff
@@ -0,0 +1,78 @@
+import reducer, { releaseState, getActions } from '@/reducers/releaseReducer';
+
+describe('releaseReducer', () => {
+  it('should return the initial state', () => {
+    const initialState = releaseState;
+    const action = { type: 'UNKNOWN_ACTION' };
+    expect(reducer(initialState, action)).toEqual(initialState);
```

Comment:
```suggestion

    expect(reducer(initialState, action)).toEqual(initialState);
```

Here for example you can add on breakline it's will give some aire to the code. And that allow to see step on the test 

1. const
2. run
3. assert (expect)